bindings/python_cffi/zyre_cffi/_cdefs.inc

```diff
@@ -93,14 +93,6 @@ void
 int
     zyre_set_endpoint (zyre_t *self, const char *format, ...);
 
-//
```

Comment:
bleh. i thought i had cleaned that out... still going through it and am spot-checking to make sure i'm not polluting more stuff :(